src/pages/lessons/[slug].tsx

```diff
@@ -199,6 +194,8 @@ const Lesson: React.FC<LessonProps> = ({
 
   const [lessonState, send] = state
 
+  const lesson: any = lessonState.context.lesson
```

Comment:
using `any` here feels bad

trusting `lessonState` for the current lesson feels good.

why would we look to the player state as a source of truth for the lesson?